docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: '3.8'
+
+services:
+  db:
+    image: postgres:15-alpine
+    ports:
+      - "5432:5432"
+    volumes:
+      - postgres_data:/var/lib/postgresql/data/
+    env_file:
+      - .env
+
+  redis:
+    image: redis:latest
+    ports:
+      - "6379:6379"
+
+  celery:
+    build:
+      context: .
+      dockerfile: Dockerfile.celery
+    volumes:
+      - .:/code
+    depends_on:
+      - db
+      - redis
+    env_file:
+      - .env
+    command: sh -c 'celery -A junction worker -l info -E'
+
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - .:/code
+    ports:
+      - "8888:8888"
+    depends_on:
+      - db
+      - redis
+      - celery
+    env_file:
+      - .env
+    command: sh -c 'python manage.py migrate && python manage.py runsslserver 0.0.0.0:8888'
```

Comment:
```suggestion
    command: sh -c 'python manage.py migrate && python manage.py sample_data && python manage.py runsslserver 0.0.0.0:8888'
```

Reply:
sample_data command should only be run once during initial setup. Not each time. Otherwise it will keep on creating the same data multiple times. For now the solution is to run a shell inside the container after startup and run `python manage.py sample_data` If there is a better solution do let us know.